tide_landing_page.install

```diff
@@ -1111,3 +1111,70 @@ function tide_landing_page_update_8028() {
   $config->set('label', 'Form embed (Drupal)');
   $config->save();
 }
+
+/**
+ * Create field status.
+ */
+function tide_landing_page_update_8029() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_install_location = [drupal_get_path('module', 'tide_landing_page') . '/config/install'];
+  _tide_import_single_config('field.storage.paragraph.field_paragraph_status', $config_install_location);
```

Comment:
_tide_read_config() is a handy function you can use to read configs from provided locations.
IMO, _tide_import_single_config is just like its comment `Helper to import a single config item (in a quick and dirty way).`, it changes uuid. 
I recommend  to use logic like below to import configs.
```
  $configs = [
    'field.storage.paragraph.field_paragraph_status' => 'field_storage_config',
  ];
  module_load_include('inc', 'tide_core', 'includes/helpers');
  $config_location = [drupal_get_path('module', 'tide_landing_page') . '/config/install'];
  // Check if field already exported to config/sync.
  foreach ($configs as $config => $type) {
    $config_read = _tide_read_config($config, $config_location, TRUE);
    $storage = \Drupal::entityTypeManager()->getStorage($type);
    $config_entity = $storage->createFromStorageRecord($config_read);
    $config_entity->save();
  }
```

Reply:
I will change it to this way thanks :)